Roadmap.md

```diff
@@ -0,0 +1,51 @@
+# Roadmap
+
+### Master branch
+| Target Unity release  | Release timeframe | Master release tag | Toolkit release features |
+| --- | --- | --- | --- |
+| 2017.2.0 | COMPLETED       | v1.2017.2.0        | <ul><li>Updated master with Windows Mixed Reality support (xR namespace).</li><li>RI Dev_Unity_2017.2.0 into master.</li><li>Toolkit will work on both HoloLens and immersive headsets.</li><li>Windows Mixed Reality motion controller support.</li></ul>|
+| 2017.3.0              | End Dec 2017      | v1.2017.3.0        | <ul><li>360 video player prefabs.</li><li>Scriptable foveated rendering prefabs.</li></ul>|
+
+### Dev_Unity_2017.2.0 branch - COMPLETED
+
+| Target Unity release  | Release timeframe | Branch release tag | Branch pre-release features | Status |
+| --- | --- | --- | --- | --- |
+| 2017.2.1 � � � � � �  | Mid Oct 2017 � �  | v1.Dev.2017.2.1 | <ul><li>Merge branch into master.</li></ul>| COMPLETED |
+
+# Future work planning
+- Automated build for MRTK-Unity.
+- Updating landing page for the ReadMe.md to help app developers find content better.
+- Updating API documentation using tools like Doxygen and not writing them manually. This is something we will work with all of you on.
+- Update Wiki to host API documentation instead.
+- Move reusable features from [Mixed Reality Design Labs](https://github.com/Microsoft/MRDesignLabs_Unity) project into toolkit.
+- Improved samples and test cases
+
+# Release cadence
+### Master branch
+- **No direct PR's into master branch.** It will be locked and only merges from stable development branches will be taken.
+- Exception will be major bug fixes on a case by case basis.
+- Please use the [working development branch](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/Dev_Working_Branch) for daily feature work.
+- [Working development branch](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/Dev_Working_Branch) will be snapped every 3 weeks for stabilization.
+- That stabilization branch will be worked on for 1 week.
+- Only bug fixes will be accepted in the stabilization branch before being merged into master.
+
+- **Master branch releases** will align with **major Unity releases marking a release every 3 months**. Please read the [Unity roadmap](https://unity3d.com/unity/roadmap).
+- Each release will be marked as a GitHub [release tag](https://github.com/Microsoft/HoloToolkit-Unity/releases). You can consume this release by:
+	- Using the zip file available in the release notes
+	- Unity packages available in the release notes
+	- Syncing to the specific git tag you would like to lock on.
+- Release tag name convention: v1.Unity release major number.Unity release minor number.Iteration number
+	- For example: For Unity version 2017.1.0 our release tag would be **v1.2017.1.0**
```

Comment:
Do we want to get more specific with the minor versioning? 

For example the fully qualified Unity Editor Release usually inlcudes `Major.Minor.0p1` or `Major.Minor.0f3` etc

Reply:
More than what I have currently?

Reply:
I think that would be ideal, but then it adds another revision number. Should we loose the v1 prefix in favor of a minor prefix that increments for a single Unity version change?

for example:
Old:
`v1.2017.1.x`

New:
`2017.1.0f3.x`

where x is our revision number for that specific unity release.

Reply:
I'm not fond of sticking THAT close to the Unity version number. I do like the idea of dropping the "v1" (what would spark a v2?).

I prefer version numbers that are comprised of only numbers (2017.2.4.3, for example)

Reply:
1. I like the idea of dropping v1 too. It was tied to older unity naming schemes which are now obsolete.

2. I also like the idea of sticking close to the Unity version numbers as it reduces confusion in developers of what this name means. Instead of inventing a new scheme ourselves, let's reuse what Unity has.

3. For minor release, I agree on the proposal. The x in the old one was also for that. However, we should try to avoid as many minors as we can to avoid too much package goop for developers. It's hard to keep track of Unity release and to add to it toolkit releases. My mind just exploded :)

I will update the PR accordingly. Thanks for the suggestions!